SU2_CFD/src/output_structure.cpp

```diff
@@ -8038,12 +8038,12 @@ void COutput::SpecialOutput_SpanLoad(CSolver *solver, CGeometry *geometry, CConf
         if (iSection == 0) {
           Dihedral_Leading = atan((Zcoord_LeadingEdge_ - Zcoord_LeadingEdge) / (Ycoord_LeadingEdge_ - Ycoord_LeadingEdge))*180/PI_NUMBER;
           Dihedral_Trailing = atan((Zcoord_TrailingEdge_ - Zcoord_TrailingEdge) / (Ycoord_TrailingEdge_ - Ycoord_TrailingEdge))*180/PI_NUMBER;
-          Dihedral = 0.5*(Dihedral_Leading + Dihedral_Trailing);
+     //   Dihedral = 0.5*(Dihedral_Leading + Dihedral_Trailing);
         }
         else {
           Dihedral_Leading = atan((Zcoord_LeadingEdge - Zcoord_LeadingEdge_) / (Ycoord_LeadingEdge - Ycoord_LeadingEdge_))*180/PI_NUMBER;
           Dihedral_Trailing = atan((Zcoord_TrailingEdge - Zcoord_TrailingEdge_) / (Ycoord_TrailingEdge - Ycoord_TrailingEdge_))*180/PI_NUMBER;
-          Dihedral = 0.5*(Dihedral_Leading + Dihedral_Trailing);
+     //   Dihedral = 0.5*(Dihedral_Leading + Dihedral_Trailing);
```

Comment:
Comment intended?

Reply:
Same here. This is to avoid a compiler warning, which was merged in earlier and reverted again. Again, it can be removed.

Reply:
Great. Can you please remove these extra commented lines to keep it clean? Then it's all good to merge.